aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
```

Comment:
BTW: this is confusing since the ASP.Net Reverse Proxy is in my mind "middleware" as well.